### PR TITLE
A4A: Update the contact support view to handle the Pressable product

### PR DESCRIPTION
**Pull request by cleacos**
Resolves: https://github.com/Automattic/automattic-for-agencies-dev/issues/896

## Proposed Changes

image

**Tested by Pressable:** The Pressable support team confirmed they received the email, one for Sales and another for Support.

## Why are these changes being made?

*

## Testing Instructions

- You have to apply this diff: D161044 
- Follow the instructions on that diff.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/a8c-for-agencies/components/user-contact-support-modal-form/index.tsx [hunk 1]
```diff
@@ -199,39 +217,48 @@ export default function UserContactSupportModalForm( {
...
-										),
-									},
-								}
+								'Your request will be routed directly to a Pressable support specialist to chat about your needs.'
 							) }
-						</p>
-					) }
+						</div>
+					</>
+				) : null }
```
Comment (jkguidaven) [inline client/a8c-for-agencies/components/user-contact-support-modal-form/index.tsx]: Nitpicking: We can skip elvis operator here:

```

				{ isPressableSelected && (
					<>
						<FormFieldset>
							<FormLabel htmlFor="product">
								{ translate( 'Would you like help with Pressable sales or support?' ) }
							</FormLabel>
							<FormSelect
								name="pressable_contact"
								id="product"
								value={ pressableContactType }
								onChange={ onPressableContactTypeChange }
							>
								<option value="sales">{ translate( 'Sales' ) }</option>
								<option value="support">{ translate( 'Support' ) }</option>
							</FormSelect>
						</FormFieldset>
						<div className="form-field-description">
							{ translate(
								'Your request will be routed directly to a Pressable support specialist to chat about your needs.'
							) }
						</div>
					</>
				) }
```

**Review by jkguidaven — APPROVED** [review]
Tested it and works great! I added 1 non-blocking comment but overall LGTM 👍

File: client/a8c-for-agencies/components/user-contact-support-modal-form/index.tsx (hunk 1 above)
Comment (cleacos) [reply client/a8c-for-agencies/components/user-contact-support-modal-form/index.tsx]: Yep! Thank you for pointing it out, much better.

Comment (cleacos) [reply client/a8c-for-agencies/components/user-contact-support-modal-form/index.tsx]: https://github.com/Automattic/wp-calypso/pull/94366/commits/1be27eb5317c89d4725c25eacd571b50221b4570

**Review by jkguidaven — APPROVED** [review]
Retested and can confirm the Pressable ID is also sent.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/16706324

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @cleacos for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16706324).

**Pull request merged**